server/shared/src/main/scala/org/http4s/server/middleware/ResponseLogger.scala

```diff
@@ -59,6 +61,7 @@ object ResponseLogger {
   )(implicit G: MonadCancelThrow[G], F: Async[F]): Kleisli[G, A, Response[F]] = {
     val fallback: String => F[Unit] = s => logger.info(s).to[F]
     val log = logAction.fold(fallback)(identity)
+    val errLog: (String, Throwable) => F[Unit] = (msg, th) => errorLogger.error(th)(msg).to[F]
```

Comment:
Perhaps a better approach would be to add `errLogAction` alongside `logAction`, allowing users to decide how to log that case. But unfortunately, that would require duplicating a bunch of constructors to preserve binary compatibility, which I’m confident would hurt ergonomics for very little gain. Maybe I’ll revisit this in 1.0, but definitely not in 0.23.